addons/pharma/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -152,6 +152,18 @@ class ACE_Medical_Treatment_Actions {
         callbackSuccess = QFUNC(medication);
         sounds[] = {};
     };
+    class TXAAuto: Morphine {
+        displayName = CSTRING(Take_TXAAuto);
+        displayNameProgress = CSTRING(Using);
+        allowedSelections[] = {"LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 1;
+        medicRequired = QGVAR(medLvl_TXAAuto);
+        items[] = {"kat_TXAAuto"};
+        condition = "";
+        treatmentTime = QGVAR(treatmentTime_TXAAuto);
+        callbackSuccess = QFUNC(medication);
+        sounds[] = {};
```

Comment:
we can use morphine injection for this sound so if you are okay with that, just delete this entry too